File: 1010.c
/*
    @autor: Guiventura6;
    @data: 20/01/2022;
    @Problema: Simple Calculate;
*/
#include <stdio.h>

int main(){
    
    int codePoduct1, codePoduct2, unitP1, unitP2;
    double priceP1, priceP2, valueToPay;

    scanf("%d%d%lf", &codePoduct1, &unitP1, &priceP1);
    scanf("%d%d%lf", &codePoduct2, &unitP2, &priceP2);

    valueToPay = unitP1 * priceP1 + unitP2 * priceP2;

    printf("VALOR A PAGAR: R$ %.2lf\n", valueToPay);

    return 0;
}

File: python/1010.py
''' 
    Autor: Guiventura6;
    Data: 01/03/2022;
    Problema: Simple Calculate;
    Resume: Program that reads data from two products. Calculates and shows the amount
    to be paid.
'''
def calculate_amount(product):
    product = tuple(map(float, product.split()))# converting string to tuple
    code, units, price = product # unpacking 
    return units * price

product1 = input()
product2 = input()

amount = calculate_amount(product1) + calculate_amount(product2)

print('VALOR A PAGAR: R$ {:.2f}'.format(amount))

